packages/terra-clinical-header/tests/jest/Header.test.jsx

```diff
@@ -6,45 +6,52 @@ it('should render a default component', () => {
   expect(header).toMatchSnapshot();
 });
 
-it('should render a header with title only', () => {
-  const header = render(<Header title="title" />);
+it('should render a header with title and heading level', () => {
+  const header = render(<Header id="test-id" level={1} text="title" />);
   expect(header).toMatchSnapshot();
 });
 
 it('should render a header with content on the left', () => {
-  const header = render(<Header startContent={<div>start content</div>} />);
+  const header = render(<Header level={1} startContent={<div>start content</div>} />);
   expect(header).toMatchSnapshot();
 });
 
 it('should render a header with content on the right', () => {
-  const header = render(<Header endContent={<div>end content</div>} />);
+  const header = render(<Header level={1} endContent={<div>end content</div>} />);
   expect(header).toMatchSnapshot();
 });
 
 it('should render a header with all content', () => {
   const header = render((
     <Header
       startContent={<div>start content</div>}
-      title="Title"
+      text="Title"
       endContent={<div>end content</div>}
+      level={1}
     />
   ));
   expect(header).toMatchSnapshot();
 });
 
-it('should render a subheader with title only', () => {
-  const subheader = render(<Header title="title" isSubheader />);
+it('should render a subheader with title and heading level', () => {
+  const subheader = render(<Header level={1} text="title" isSubheader />);
   expect(subheader).toMatchSnapshot();
 });
 
 it('should render a subheader with all content', () => {
   const subheader = render((
     <Header
       startContent={<div>start content</div>}
-      title="Title"
+      text="Title"
       endContent={<div>end content</div>}
       isSubheader
+      level={1}
     />
   ));
   expect(subheader).toMatchSnapshot();
 });
+
+it('should render a header with no text from level not set', () => {
+  const header = render(<Header text="This title should not render without a provided level" />);
+  expect(header).toMatchSnapshot();
+});
```

Comment:
We should explicitly test for the value in addition to the snapshot.

Reply:
This was added to validate non-passive change on it's previous PR. now with Passive change we still render header with title even when value is not set for prop `level`. It uses default level h1 to maintain passivity.

Updated the test to validate console warning thrown when value for `level` prop is not set https://github.com/cerner/terra-clinical/pull/823/commits/2a0747c1873dccb3053cc005b59cdea87205589c#diff-2a241fab4e9f4e0d99b828bc9f8e3a451b22d15bcb991b77784d45ceb03af1de